inbox/api/jsc_api.py

```diff
@@ -115,15 +107,13 @@ def enable_sync():
             account = namespace.account
             account.sync_state = 'running'
             account.sync_should_run = True
-            account.sync_host = '{}:{}'.format(platform.node(), 0)
 
             if account.provider == 'gmail':
                 creds = account.auth_credentials
                 for c in creds:
                     c.is_valid = True
 
             db_session.commit()
-            notify_node()
```

Comment:
just to confirm: on line 86 you manually sent a sync_suspended event but here we don't need to send an event?

Reply:
No, it is sent automatically by the account model (see https://github.com/jobscore/sync-engine/blob/feature/scale/inbox/models/account.py#L353)